src/main/python/design_view.py

```diff
@@ -80,17 +70,24 @@ def __init__(self, main_window, *args, **kwargs):
         self.cancel_actions_shortcut.activated.connect(self.cancel_actions)
 
     def save_model(self):
```

Comment:
The naming scheme `save_model` etc. might be confused for the NLP models. Maybe we should use something more descriptive like "taskboard" or something? Your thoughts Alex?

Reply:
Yeah, so far I have just called it a process but taskboard is not a bad idea. I was just using model as in data model.